### PR TITLE
Waiting page

### PR DESCRIPTION
**Pull request by anivm01**
waiting page, some styling on frontpage, username prop passed from app.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import RegistrationPage from './Components/RegistrationPage.js'
+import './App.css';
+import FrontPage from './Components/FrontPage.js';
+import { useState } from 'react';
+import WatiningPage from './Components/WatiningPage.js';
```
Comment (andrewtrac) [inline src/App.js]: Watining

File: src/App.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+  const [userName, setUserName] = useState("")
+  const [userEmail, setUserEmail] = useState("")
   return (
     <BrowserRouter>
     <Routes>
-      <Route path="registration" element={<RegistrationPage userEmail={userEmail} />} />
-      <Route path='match' element={<ProfilePage/>} />
-      <Route path='/' element={<FrontPage setUserEmail={setUserEmail}/>} />
-      <Route path="biopage" element={<BioPage />} />
+      <Route path='/' element={<FrontPage setUserEmail={setUserEmail} setUserName={setUserName}/>} />
```
Comment (andrewtrac) [inline src/App.js:16]: Seems to have props for setUserName and setUserEmail but only one of them is used

File: src/Components/FrontPage.js [hunk 3]
```diff
@@ -1,32 +1,44 @@
 import { useState } from "react"
 import TagLines from "./TagLines"
+import "./FrontPage.css"
+import { useNavigate } from "react-router-dom"
 
 function FrontPage( {setUserEmail} ) {
+    const [name, setName] = useState("")
```
Comment (andrewtrac) [inline src/Components/FrontPage.js:7]: These two setstates shouldn't be used if you're passing them down from props

Comment (anivm01) [reply src/Components/FrontPage.js:7]: The set name is to control the input

Comment (anivm01) [reply src/Components/FrontPage.js:7]: setUserName is to pass the info between components

**Review by andrewtrac — APPROVED** [review]

**Pull request merged**